lib/active_model/serializer.rb

```diff
@@ -19,7 +19,7 @@
 module ActiveModel
   class Serializer
     # @see #serializable_hash for more details on these valid keys.
-    SERIALIZABLE_HASH_VALID_KEYS = [:only, :except, :methods, :include, :root].freeze
+    SERIALIZABLE_HASH_VALID_KEYS = [:only, :except, :methods, :include].freeze
```

Comment:
root still belongs here, though technically it is an option for `as_json`. This  isn't about partitioning and need  not be changed. In fact, it's an option in the ActiveModel::Serializers::JSON interface